dashboard/src/routes/issue/route.tsx

```diff
@@ -0,0 +1,30 @@
+import { createFileRoute, stripSearchParams } from '@tanstack/react-router';
+
+import { z } from 'zod';
+
+import {
+  makeZIntervalInDays,
+  zListingSize,
+  type SearchSchema,
+} from '@/types/general';
+import {
+  DEFAULT_LISTING_ITEMS,
+  DEFAULT_TIME_SEARCH,
+} from '@/utils/constants/general';
+
+const defaultValues = {
+  intervalInDays: DEFAULT_TIME_SEARCH,
+  issueSearch: '',
+  listingSize: DEFAULT_LISTING_ITEMS,
+};
+
+const zIssueSearchSchema = z.object({
+  intervalInDays: makeZIntervalInDays(DEFAULT_TIME_SEARCH),
+  issueSearch: z.string().catch(''),
+  listingSize: zListingSize,
+} satisfies SearchSchema);
```

Comment:
```suggestion
const issuesListingDefaultValues = {
  intervalInDays: DEFAULT_TIME_SEARCH,
  issueSearch: '',
  listingSize: DEFAULT_LISTING_ITEMS,
};

const issuesListingSearchSchema = z.object({
  intervalInDays: makeZIntervalInDays(DEFAULT_TIME_SEARCH),
  issueSearch: z.string().catch(''),
  listingSize: zListingSize,
} satisfies SearchSchema);
```

to maintain consistent with other routes files (see build or test route files).

Reply:
I'd say we only change these variable names if we are exporting them, otherwise there's no need. (All other routes besides build or test have these named as `defaultValues` and `z<name>SearchSchema

Reply:
Since I'll make the shortcut route, I'll export them, so I will rename the variables too